### PR TITLE
Lg/dcos 55145 edgelb diag

### PR DESCRIPTION
**Pull request by lsgunnlsgunn**
## Jira Ticket
Before creating this pull request, please make sure you have a separate ticket for the docs team to track their work to review and merge, even for minor edits. If you need assistance, ask in the #doc-team channel on slack.
https://jira.mesosphere.com/browse/DCOS-55145

## Description of changes being made
DRAFT command reference for 1.4 Edge-LB diagnostic command. The same information is being added to the next generation version of the guide not yet published.

## Checklist
- [ ] Make sure you change all affected versions (e.g. 1.10, 1.11, 1.12, 1.13).
- [ ] Test all commands and procedures where applicable.
- [ ] Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects) if you are moving a page.

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing) for more information.

### CODE REVIEW

File: pages/services/edge-lb/1.4/cli-reference/dcos-edgelb-cleanup/index.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+layout: layout.pug
+navigationTitle:  dcos edgelb cleanup
+title: dcos edgelb cleanup
+menuWeight: 5
+excerpt: List and remove elements of the AWS Elastic Load Balancer (ELB) framework created by Edge-LB
+enterprise: true
+---
+
+# Description
+The `dcos edgelb cleanup` command lists and removes all Amazon Web Services (AWS) Elastic Load Balancer (FLB) instances that remain after Edge-LB has been uninstalled from a DC/OS cluster. This command also uninstalls the Edge-LB cloud controller load balancer framework, including the cloud controller listener and target group.
```
Comment (GoelDeepak) [inline pages/services/edge-lb/1.4/cli-reference/dcos-edgelb-cleanup/index.md]: typo FLB

**Comment by lsgunnlsgunn** [comment]
Pulled typo in from another branch. Close this open and open a new unified PR.

File: pages/services/edge-lb/1.4/cli-reference/dcos-edgelb-diagnostic/index.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+To collect diagnostic bundles for all Edge-LB pools, run the following command:
+
+```bash
+dcos edgelb diagnostic
+```
+
+To collect diagnostic bundles for specific Edge-LB pools, include the pool names in a command similar to the following:
+
+```bash
+dcos edgelb diagnostic --pool-names=sf-edgelb, roma-edge-lb, hk-edgelb
```
Comment (mainred) [inline pages/services/edge-lb/1.4/cli-reference/dcos-edgelb-diagnostic/index.md]: There are no spaces
```suggestion
dcos edgelb diagnostic --pool-names=sf-edgelb,roma-edge-lb,hk-edgelb
```

File: pages/services/edge-lb/1.4/cli-reference/dcos-edgelb-diagnostic/index.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+```
+
+This command generates diagnostic bundle with the logs files from the `sf-edgelb`, `roma-edgelb`, and `hk-edgelb` pools.
```
Comment (mainred) [inline pages/services/edge-lb/1.4/cli-reference/dcos-edgelb-diagnostic/index.md]: and placed in the current working directory.

File: pages/services/edge-lb/1.4/cli-reference/dcos-edgelb-diagnostic/index.md (hunk 2 above)
Comment (lsgunnlsgunn) [reply pages/services/edge-lb/1.4/cli-reference/dcos-edgelb-diagnostic/index.md]: Fixed in SAVE-EDGE-LB-1.4 working branch.

File: pages/services/edge-lb/1.4/cli-reference/dcos-edgelb-diagnostic/index.md (hunk 3 above)
Comment (lsgunnlsgunn) [reply pages/services/edge-lb/1.4/cli-reference/dcos-edgelb-diagnostic/index.md]: Fixed in SAVE-EDGE-LB-1.4 working branch.

**Pull request closed without merging**